pkgdb2/forms.py

```diff
@@ -127,11 +139,11 @@ class AddPackageForm(wtf.Form):
         'Summary',
         [wtforms.validators.Required()]
     )
-    description = wtforms.TextField(
+    description = wtforms.TextAreaField(
         'Description',
     )
     review_url = wtforms.TextField(
-        'Review URL',
+        'Review URL <span class="error">*</span>',
```

Comment:
Is this ``<span>`` not added automatically by ``wtforms`` when you add the ``wtforms.validators.Required()``?